### PR TITLE
Moved examples and templates to from app to docs

### PR DESCRIPTION
**Pull request by davidhunter08**
- moved examples and templates from app to docs
- updated documentation_routes.js
- deleted layout.html from app folder
- updated templates, passing data and branching guidance

### CODE REVIEW

File: app/views/content-page.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {% extends 'layouts/layout-nhsuk.html' %}
```
Comment (chrimesdev) [inline app/views/content-page.html]: There seems to be 2 `content-page` with different content

One at `/templates/content-page` (docs/views/templates/content-page.html) which is linked to from the page templates and one at `/templates/content-page` (app/views/content-page) which is linked to from the non-docs website homepage - this page seems to have better content. They used to have the same content but have diverged, do we need both?

File: app/views/includes/scripts.html [hunk 2]
```diff
@@ -1,5 +1,4 @@
 <script src="/js/main.js"></script>
 <script src="/js/jquery-3.3.1.min.js"></script>
-<script src="/js/auto-store-data.js"></script>
```
Comment (chrimesdev) [inline app/views/includes/scripts.html:3]: Is this the old method of passing data? Has it been replaced by another method?

File: docs/views/templates.html [hunk 3]
```diff
@@ -18,7 +18,7 @@
 
       <h1>Page templates</h1>
 
-      <p>You can find these templates in the <strong>app/views/templates</strong> folder.</p>
+      <p>You can find these templates in the <strong>doc/views/templates</strong> folder.</p>
```
Comment (chrimesdev) [inline docs/views/templates.html]: Should be `docs` not `doc`

**Review by chrimesdev — CHANGES_REQUESTED** [review]
Noticed a few changes needed.

Also need to bump to version in `package.json` and `package-lock.json`, with a CHANGELOG entry with an explanation of the changes.

File: app/views/index.html [hunk 4]
```diff
@@ -28,19 +28,36 @@ <h1>
...
+      </h2>
 
-      <p>Use this page as the index for your project.</p>
+      <p>
+        Use this page as the index for your project.
+      </p>
       
-      <p>You'll find this page in the kit at <strong>/app/views/index.html</strong>.</p>
+      <p>
+        The code for this page in the kit, you can find it at <strong>/app/views/index.html</strong>.
```
Comment (chrimesdev) [inline app/views/index.html]: Does this sentence read properly?

Comment (chrimesdev) [inline app/views/index.html]: For consistency... when telling users to look in a folder, we've used '/' before the folder here, however on other examples such as on the `docs/views/how-tos/passing-data-page.html` page. There is no '/' before the folder 'docs'.

File: docs/views/how-tos/branching.html [hunk 5]
```diff
@@ -21,7 +21,7 @@ <h1>How to do branching</h1>
 
       <p>The code for the example can be found in:</p>
 
-      <pre class="app-pre"><code class="app-code">/documentation_routes.js<br>/views/examples/branching</code></pre>
+      <pre class="app-pre"><code class="app-code">/documentation_routes.js<br>docs/views/examples/branching</code></pre>
```
Comment (chrimesdev) [inline docs/views/how-tos/branching.html:24]: See comment here: https://github.com/nhsuk/nhsuk-prototype-kit/pull/96/files#r266793931

File: app/views/content-page.html (hunk 1 above)
Comment (davidhunter08) [reply app/views/content-page.html]: We need both, I have renamed the one in app to page-example.html to save confusion.

File: docs/views/templates.html (hunk 3 above)
Comment (davidhunter08) [reply docs/views/templates.html]: Done

File: app/views/index.html (hunk 4 above)
Comment (davidhunter08) [reply app/views/index.html]: I have updated the content.

Comment (davidhunter08) [reply app/views/index.html]: I have added a '/' for all.

**Review by chrimesdev — APPROVED** [review]

**Pull request merged**